app/models/functionaris.js

```diff
@@ -1,6 +1,7 @@
 import Model, { attr, belongsTo } from '@ember-data/model';
+import MandatarisModel from './mandataris';
 
-export default class FunctionarisModel extends Model {
+export default class FunctionarisModel extends MandatarisModel {
```

Comment:
why did you do this? and don't we need to set polymorphic on some relationships then?